server.js

```diff
@@ -15,6 +15,7 @@ const server = httpServer({
     }
   },
 });
+const timestamps = new Date().toISOString();
```

Comment:
@arafatalim Once server is started, this time will be set and it won't change as it is defined as constant, so for every api call same time will be logged, please change it.

Reply:
done!